<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="backtrackCheckBox">
     <property name="text">
      <string>Backtrack</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel">
     <property name="text">
      <string>Fit method:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="methodMatchButton">
     <property name="text">
      <string>MATCH</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_method</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="methodORMButton">
     <property name="text">
      <string>ORM</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_method</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="methodSectormapButton">
     <property name="text">
      <string>sectormap</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttongroup_method</string>
     </attribute>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttongroup_method"/>
 </buttongroups>
</ui>
